deparse.c

```diff
@@ -3907,3 +3954,19 @@ sqlite_classify_conditions(PlannerInfo *root,
 			*local_conds = lappend(*local_conds, ri);
 	}
 }
+
+/*
+ * Case independend string comparation is usrful for text option values is some cases
+ */
+int strcmp_case_independend(const char *str1, const char *str2) {
```

Comment:
I think you can use `strcasecmp` instead of create new function.

Reply:
Thanks, replaced. `strcasecmp` works fine.

Reply:
Thank for updating. Confirmed.